bc_obps/registration/api/operator.py

```diff
@@ -81,4 +87,21 @@ def list_user_operators_status_of_operator(request, operator_id: int):
 ##### PUT #####
 
 
+@router.put("/operators/{operator_id}", response={200: OperatorOut, codes_4xx: Message})
+def update_operator(request, operator_id: int, payload: OperatorIn):
+    raise_401_if_role_not_authorized(request, ["cas_admin", "cas_analyst"])
+    operator = get_object_or_404(Operator, id=operator_id)
+    user: User = request.current_user
+    operator.status = payload.status
+    if operator.status in [Operator.Statuses.APPROVED, Operator.Statuses.REJECTED]:
```

Comment:
Re this AC: if the internal user Requests Changes for the new Operator, the is_new flag will remain as True until the Operator's status is changed to Approved

`is_new` defaults to true, so we don't have to do anything besides not include the CHANGES status to implement this AC